project/settings/development.py

```
@@ -0,0 +1,24 @@
+INSTALLED_APPS += [
+    "django_browser_reload",
+]
+
+MIDDLEWARE += [
+    "django_browser_reload.middleware.BrowserReloadMiddleware",
+]
+
+DATABASES = {
+    "default": {
+        "ENGINE": "django.db.backends.sqlite3",
+        "NAME": BASE_DIR / "db.sqlite3",
```

Ruff (0.8.2)

`BASE_DIR` may be undefined, or defined from star imports

(F405)